platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -470,13 +442,13 @@ private RuntimeWorkspaceDto asDto(RuntimeWorkspace workspace) {
                                                 .withEnvironments(usersWorkspace.getEnvironments())
                                                 .withAttributes(workspace.getAttributes())
                                                 .withActiveEnvName(workspace.getActiveEnvName())
-                                                .withDevMachine(asDto(workspace.getDevMachine()))
+                                                .withDevMachine(asRuntimeWorkspaceDto(workspace.getDevMachine()))
                                                 .withRootFolder(workspace.getRootFolder())
                                                 .withMachines(machines)
                                                 .withLinks(links);
     }
 
-    private MachineDto asDto(Machine machine) {
+    private MachineDto asRuntimeWorkspaceDto(Machine machine) {
```

Comment:
I decided to move to `DtoConverter` only methods which are need for `AccountService`. 
It is kind of temporary solution which will be replaced with _Registry_ for DTO transformers [asap](https://github.com/codenvy/che-core/blob/IDEX-2812/platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/DtoConverter.java#L34).
I think [model mapper](http://modelmapper.org/) is good enough for it, but it worth some time(which i don't have) to investigate it.